cumulusci/core/flowrunner.py

```diff
@@ -628,33 +630,37 @@ def _check_old_yaml_format(self):
             else:
                 raise FlowConfigError("No steps found in the flow definition")
 
-    def _check_infinite_flows(self, flow_config, visited_flows=None):
+    def _check_infinite_flows(self, flow_config, flow_stack=None):
         """
         Recursively loop through the flow_config and check if there are any cycles.
 
-        :param steps: Set of step definitions to loop through
-        :param flows: Flows already visited.
+        :param flow_config: Set of step definitions to loop through
```

Comment:
This isn't actually a `Set`, it's a `FlowConfig` object (which we want to ensure doesn't have cycles/infinite loops).